runtime/kusama/src/lib.rs

```diff
@@ -1704,6 +1704,10 @@ sp_api::impl_runtime_apis! {
 			Err(mmr::Error::PalletNotIncluded)
 		}
 
+		fn mmr_leaves_count() -> Result<mmr::LeafIndex, mmr::Error> {
+			Err(mmr::Error::PalletNotIncluded)
+		}
+
```

Comment:
I believe all the runtimes (kusama, polkadot, rococo, westend, etc) need to bump RuntimeAPI version number when adding this new API method.

Reply:
As discussed and tried offline, in case the method is not present in the API we get an Application(Execution(Other("Exported method MmrApi_mmr_leaves_count is not found"))) . This is treated by the mmr gadget code, a debug log is printed and the loop that checks for finality notifications continues to run. So we don't need to change the Runtime API version.